packages/fxa-content-server/webpack.config.js

```diff
@@ -34,7 +34,11 @@ const webpackConfig = {
       'webrtc',
       'styles/main.scss'
     ],
-    head: './head/boot.js'
+    head: './head/boot.js',
+    payment: './payment/app.js',
+    paymentDependencies: [
```

Comment:
I played around with separating out dependencies, but honestly couldn't figure out what was going on. For instance with this one dependency (which is required by `config-loader`), the resulting file doesn't actually contain any real code! I'm very confused. The `payment.bundle.js` file does contain many modules, but has the loading problem I mention earlier.

Reply:
I think the way [`optimization.splitChunks.cacheGroups` is configured later on in the file](https://github.com/mozilla/fxa/blob/25d899a69f7c89c27b41514b8ce37205ba9ec2a1/packages/fxa-content-server/webpack.config.js#L155), any dependencies already showing up in `appDependencies` will be omitted from every other entry - including this new `paymentDependencies` entry.

If you use `appDependencies.bundle.js` instead of `paymentDependencies.bundle.js` in `server/templates/pages/src/payment.html`, things start working. What I think is happening otherwise is that the `payment.bundle.js` bundle just sits waiting forever for the async loading of modules expected from `appDependencies`.

We probably do want "duplicate" dependencies in bundles here, since they'll be served on different domains. I think there is a way to do that, but it's been awhile since I last hacked around with vendor bundle splitting in webpack.